src/providers/authn.tsx

```diff
@@ -170,31 +170,29 @@ export default function AuthnProvider({ children }: AuthnProviderProps): JSX.Ele
       };
     }
 
-    // make sure the width and height are not bigger than the screen
     const popupWidth = Math.min(800, screen.availWidth);
-    const popupHeight = Math.min(750, screen.availHeight)
-    const topOffset = 50;
+    const popupHeight = Math.min(750, screen.availHeight);
 
-    // left should be in the middle of the screen
-    const popupLeft = (screen.availWidth - popupWidth) / 2;
-    // top should just have some small offset if there's available space
-    const popupTop = (topOffset + popupHeight) < screen.availHeight ? topOffset : 0;
+    const primaryScreen = window.screen;
 
+    // Calculate the center position based on the primary screen dimensions and popup size
+    const centerLeft = Math.max(0, (primaryScreen.availWidth - popupWidth) / 2 + primaryScreen.availLeft);
+    const centerTop = Math.max(0, (primaryScreen.availHeight - popupHeight) / 2 + primaryScreen.availTop);
```

Comment:
Let's leave this to what it was. We still want the popup to show up closer to the top of the page, so it's more visible.

Reply:
reverted